app/models/user.rb

```diff
@@ -25,7 +25,11 @@ class User < ApplicationRecord
   validates :email, presence: true, uniqueness: { case_sensitive: false, scope: :provider }
   validates :provider, presence: true
 
-  validates :password, presence: true, confirmation: true, on: :create, unless: :external_id?
+  validates :password,
+            presence: true,
+            format: /\A(?=.*[a-z])(?=.*[A-Z])(?=.*\d)(?=.*(\W|_)).{8,}\z/,
```

Comment:
Should we add a comment that explains what the format needs to be so it's easier to read :
`Atleast 1 uppercase, Atleast 1 lowercase, Atleast 1 symbol, Atleast 1 number`

Reply:
Will do it in  follow up pr